src/components/pages/PowerheadPage/view.js

```diff
@@ -69,32 +72,56 @@ function renderFinancialsStatsList(report, monthIndex) {
   const revenueBarStyle = styles.financialsStatsLegendBarRevenue;
   return (
     <ul className={styles.financialsStats}>
-      {renderFinancialsStatsItem('Overruns', overruns, overrunsBarStyle, true)}
-      {renderFinancialsStatsItem('Confirmed revenue', revenue, revenueBarStyle)}
+      {renderFinancialsStatsItem(
+        'Overruns', overruns, overrunsBarStyle, 'overruns', true
+      )}
+      {renderFinancialsStatsItem(
+        'Confirmed revenue', revenue, revenueBarStyle, 'confirmed_revenue'
+      )}
     </ul>
   );
 }
 
 function renderMonthGraphPeople(report, monthIndex) {
+  const useSmallBoxesThreshold = 52;
```

Comment:
Not for this PR, but I think a better approach would be to dynamically stretch in and out, instead of a magic-number threshold approach. E.g. If a tribe suddenly has 53 people (quite possible), it would suddenly look not so nice.